mate_bot/commands/pay.py

```diff
@@ -41,7 +41,12 @@ def __str__(self):
 class PayCommand(BaseCommand):
 
     def __init__(self):
-        super().__init__("pay", "")
+        super().__init__(
+            "pay",
+            "The `/pay` command enables users who spend money for the community to get "
+            "their money back. Using `/pay <amount> <reason>` you can claim you bought "
+            "something for the community. Others have to approve before the money goes "
+            "to your account.")
```

Comment:
`the money goes to your account` seems to be pretty bad English style, doesn't it? 🤔 

Please expand this description a little bit, even though the implementation of the command wasn't started yet. As it's relatively similar, you can look into the description of the `/communism` command as well.